### PR TITLE
Adds CookiePath to SessionDefaults

### PR DESCRIPTION
**Pull request by hishamco**
Remove the duplication in SessionOptions, SessionMiddleware [Refactoring], by adding a "CookiePath" to the "SessionDefaults"

### CODE REVIEW

File: src/Microsoft.AspNet.Session/SessionDefaults.cs [hunk 1]
```diff
@@ -8,5 +8,6 @@ namespace Microsoft.AspNet.Session
     public static class SessionDefaults
     {
         public static string CookieName = ".AspNet.Session";
+		public static string CookiePath = "/";
```
Comment (Eilon) [inline src/Microsoft.AspNet.Session/SessionDefaults.cs]: Can you switch the tab to spaces?

**Comment by Eilon** [comment]
Overall looks good, just one comment. Also, can you squash the commits into one commit?

File: src/Microsoft.AspNet.Session/SessionDefaults.cs (hunk 1 above)
Comment (hishamco) [inline src/Microsoft.AspNet.Session/SessionDefaults.cs]: Of course @Eilon I thought the tab will works fine

**Comment by hishamco** [comment]
Sorry @Eilon for inconvenience i faced an issues with switching between VS & GitHub Viewer, but finally I squashed the commits into a single one

**Comment by Eilon** [comment]
Looks good!

**Pull request merged**